### ISSUE TITLE
 mb. PLAIN TEXT TRACKLIST does no longer appear

### ISSUE THREAD

**Issue by kellnerd** [jesus2099/konami-command#619]
Probably another of these React redraw issues on the new release pages 😢

**Comment by jesus2099** [comment]
I will implement web service mode (#65) once I can make it work for my collection script (#64).

**Comment by jesus2099** [comment]
Hi @kellnerd, do you still need this? 

I think I don't really miss this script, myself, but maybe you need one specific format, that's different from MusicBrainz track parser format?

**Comment by kellnerd** [comment]
Probably not, since I haven't used it for three years. You can close this issue if the feature is unlikely to come back.

I don't even remember my exact workflow back then, but I seem to recall that I used the track parser format and it was just convenient to save a few clicks (Edit release, tracklist tab, open tracklist parser) when copying a tracklist.

**Comment by jesus2099** [comment]
Yes, it saved a little time for me too, but I think now not enough to justify fixing it.
It needs big fixing for [big releases](https://github.com/jesus2099/konami-command/blob/4d6d09da544039185b01d11dc1dda77ba8636e0f/CONTRIBUTING.md#interesting-mb-test-pages)…

I think I might just as well drop this script. 😭
